pkg/controllers/federatedcluster/controller.go

```diff
@@ -267,18 +265,19 @@ func (c *FederatedClusterController) collectClusterStatus(qualifiedName common.Q
 	}()
 
 	cluster, err := c.clusterLister.Get(qualifiedName.Name)
-	if err != nil && apierrors.IsNotFound(err) {
```

Comment:
Why do we log an error when the cluster is not found?

Reply:
Misread the diff location.

Reply:
Right... Good catch.